src/Abp/Authorization/AuthorizationHelper.cs

```diff
@@ -59,6 +59,11 @@ public virtual async Task AuthorizeAsync(MethodInfo methodInfo, Type type)
 
         protected virtual async Task CheckFeatures(MethodInfo methodInfo, Type type)
         {
+            if (!AbpSession.TenantId.HasValue)
```

Comment:
This would be better:
```suggestion
            if (AbpSession.UserId.HasValue && !AbpSession.TenantId.HasValue)
```
What do you think?